soul-register-center/soul-register-client/soul-register-client-http/src/main/java/org/dromara/soul/register/client/http/utils/RegisterUtils.java

```diff
@@ -42,7 +42,7 @@ private RegisterUtils() {
     public static void doRegister(final String json, final String url, final String rpcType) {
         try {
             String result = OkHttpTools.getInstance().post(url, json);
-            if ("success".equals(result)) {
+            if ("SUCCESS".equals(result)) {
```

Comment:
Why modify this?

Reply:
@PostMapping("/springmvc-register")
    public String registerSpringMvc(@RequestBody final SpringMvcRegisterDTO springMvcRegisterDTO) {
        INSTANCE.publishEvent(RpcTypeEnum.HTTP.getName(), springMvcRegisterDTO);
        return "SUCCESS";
    }

Reply:
...I suggest that use `org.dromara.soul.admin.utils.SoulResultMessage#SUCCESS` in controller.

Reply:
public String registerSpringMvc(@RequestBody final SpringMvcRegisterDTO springMvcRegisterDTO) {
        INSTANCE.publishEvent(RpcTypeEnum.HTTP.getName(), springMvcRegisterDTO);
        return SoulResultMessage.SUCCESS;
    }

like this?

Reply:
yes. That's what I mean. thanks.

Reply:
okay，I have modified.